packages/anoncreds/tests/InMemoryAnonCredsRegistry.ts

```diff
@@ -26,8 +28,28 @@ export class InMemoryAnonCredsRegistry implements AnonCredsRegistry {
   // we want, but the indy-sdk is picky about the identifier format.
   public readonly supportedIdentifier = /^[a-zA-Z0-9]{21,22}/
 
-  private schemas: Record<string, AnonCredsSchema> = {}
-  private credentialDefinitions: Record<string, AnonCredsCredentialDefinition> = {}
+  private schemas: Record<string, AnonCredsSchema>
+  private credentialDefinitions: Record<string, AnonCredsCredentialDefinition>
+  private revocationRegistryDefinitions: Record<string, AnonCredsRevocationRegistryDefinition>
+  // { revocationRegistryDefinitionId: { timestamp: revocationStatusList }
```

Comment:
Is this commented code left for some purpose to handle later on?

Reply:
It can be removed! 👍